src/stories/Library/content-list-item/ContentListItem.tsx

```diff
@@ -30,32 +30,32 @@ export const EventListItem: React.FC<EventListItemProps> = ({
   const placeholderDateTime = "2023-03-10T15:00"; // ISO format date and time
 
   return (
-    <a href={href} className="event-list-item arrow__hover--right-small">
-      <div className="event-list-item__image-container">
-        <img src={image} alt={title} className="event-list-item__image" />
+    <a href={href} className="content-list-item arrow__hover--right-small">
+      <div className="content-list-item__image-container">
+        <img src={image} alt={title} className="content-list-item__image" />
       </div>
-      <div className="event-list-item__content">
+      <div className="content-list-item__content">
         {tagText && (
-          <Tag hasBackground className="event-list-item__tag">
+          <Tag hasBackground className="content-list-item__tag">
             {tagText}
           </Tag>
         )}
-        <div className="event-list-item__date">{date}</div>
-        <h2 className="event-list-item__title">{title}</h2>
-        <div className="event-list-item__description">
+        <div className="content-list-item__date">{date}</div>
+        <h2 className="content-list-item__title">{title}</h2>
+        <div className="content-list-item__description">
           <p>{description}</p>
         </div>
-        <div className="event-list-item__location-wrapper">
-          <p className="event-list-item__location">{location}</p>
+        <div className="content-list-item__location-wrapper">
```

Comment:
It's just an idea, but could you call content-list-item__bottom-wrapper and also have a content-list-item__right-wrapper If now the data inside changes?